rfhub2/cli/keywords_importer.py

```diff
@@ -276,6 +278,9 @@ def _serialise_args(self, args: List[str]) -> str:
             else ""
         )
 
+    def _serialise_tags(self, tags) -> str:
```

Comment:
what is the type of single tag here? if it is `str` than `str()` is not required. If it some class instance, maybe you could access some field to get tag name.
Return type should be `List[str]`

Reply:
It would be changed to:
```
from robot.model import tags

    def _serialise_tags(self, tags: tags) -> List[str]:
        return list(tags._tags)